tasks/ubuntu/bionic.task/metadata.yaml

```diff
@@ -4,3 +4,6 @@ os_version: bionic
 label: Ubuntu Bionic 18.04
 description: Ubuntu Bionic Installer
 base: ubuntu
+boot_sequence:
```

Comment:
This change shouldn't be needed. The sequence is inherited from `ubuntu.task/metadata.yaml`, which is identical to this.

Reply:
I'll remove this one.

Thanks,